Makefile-libpriv.am

```diff
@@ -23,8 +23,6 @@ librpmostreepriv_sources = \
 	src/libpriv/rpmostree-util.cxx \
 	src/libpriv/rpmostree-util.h \
 	src/libpriv/rpmostree-types.h \
-	src/libpriv/rpmostree-passwd-util.cxx \
-	src/libpriv/rpmostree-passwd-util.h \
```

Comment:
Hooray for C++ files being completely deleted! :tada:

Reply:
For stats: this was ~1370 lines of C at beginning, it's ~930 lines of Rust at the end. It surfaced at least a missing NULL check (the unhandled path resolution case in the comments in this PR).